client/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
```

Comment:
you did not import the jest testing library that could be the reason your tests are not running